src/view/OptionsMenu.py

```diff
@@ -0,0 +1,17 @@
+from typing import List, Optional, Any, Dict
+
+from pygame.event import EventType
+from pygame.surface import Surface
+
+from view.Action import Action
+from view.View import View
+
+
+class OptionsMenu(View):
+    def __init__(
+        self, available: Any, output_dataset: Dict[str, Any], next: Optional[int] = None
+    ) -> None:
```

Comment:
```suggestion
    def __init__(
        self, 
        available: Any, 
        output_dataset: Dict[str, Any], 
        next: Optional[int] = None,
    ) -> None:
```

Reply:
This should be enforced by pre-commit, but it's not. Does not affect functionality.